utils/build-script

```diff
@@ -601,6 +601,9 @@ class BuildScriptInvocation(object):
                         args.build_jobs)
                 ]
 
+        if args.enable_experimental_differentiable_programming:
+            impl_args += ["--enable-experimental-differentiable-programming"]
```

Comment:
I don't think you need to do this in build-script-impl. You should be able to add this as a cmake option like this: https://github.com/apple/swift/blob/master/utils/swift_build_support/swift_build_support/products/swift.py

Reply:
Thanks! Don in https://github.com/apple/swift/pull/27595/commits/4bef66beffe35fbe143462986391c976f4c72f98 - could you please re-review?